packages/react-native-reanimated/Common/cpp/reanimated/CSS/interpolation/groups/RecordPropertiesInterpolator.cpp

```diff
@@ -108,9 +84,8 @@ folly::dynamic RecordPropertiesInterpolator::mapInterpolators(
     const {
   folly::dynamic result = folly::dynamic::object;
 
-  for (const auto &[propName, interpolator] : interpolators_) {
-    auto value = callback(*interpolator);
-    result[propName] = value;
+  for (const auto &pair : interpolators_) {
+    result[pair.first] = callback(*pair.second);
```

Comment:
I think unpacking it directly in the `for` loop definition would make it more readable, but the performance of both approaches is the same.